### DISCUSSION TITLE
Instance var inline initialization

### DISCUSSION THREAD

**Discussion by s-hadinger** [berry-lang/berry#192] [Ideas]
I'm bringing back an old idea, ability to do:

``` ruby
class test
    var i=0
    var s="some string"
    var t=time.now()
end
```

Instead of putting all this in the `init()` method. However I didn't find a simple elegant way to do it. I believe that it implies to implement a new `preinit()`-like function that runs the initialization code first, and handling inheritance might be complex.

**Comment by skiars** [comment]
I have considered this feature, but it does not seem to be implemented.

**Reply by s-hadinger** [reply]
Understood. This is not a simple change for a single-pass compiler.
